### PR TITLE
[benchmark] Upgraded Benchmark test to support more image classification models.

### LINKED ISSUE #310: [benchmark] Update Benchmark test corresponding with examples

### PR DESCRIPTION
**Pull request by BruceDai**
This is PR is to:

1. Upgraded Benchmark test to support more image classification models. Fixes #310
2. Upgraded examples with Preference settings.

@huningxin @ibelem PTAL, Thanks.

### CODE REVIEW

File: examples/image_classification/main.js [hunk 1]
```diff
@@ -504,7 +367,7 @@ function main(camera) {
       }).catch((e) => {
         console.warn(`Failed to init ${utils.model._backend}, try to use WASM`);
         console.error(e);
-        showAlert(utils.model._backend);
+        showAlert(utils.model._backend, currentModel.modelName);
```
Comment (ibelem) [inline examples/image_classification/main.js]: For your information, i have totally refactored the codes for imageclassification main.js , and will also do the same thing for other examples.

**Comment by ibelem** [comment]
Travis CI failed again. @BruceDai

**Comment by BruceDai** [comment]
OK, I will check and re-run Travis CI. thanks

**Comment by BruceDai** [comment]
@ibelem Rerun then Travis CI passed.

**Comment by ibelem** [comment]
LGTM, thanks for the improvements! There will be upcoming updates for util/base.js...

**Pull request merged**